src/components/media-views.js

```diff
@@ -545,26 +549,39 @@ AFRAME.registerComponent("media-video", {
   },
 
   tick() {
+    if (!this.video) return;
+
     const userinput = this.el.sceneEl.systems.userinput;
     const volumeMod = userinput.get(paths.actions.cursor.mediaVolumeMod);
-    if (volumeMod) {
+    if (this.el.is("hovered") && volumeMod) {
       this.el.setAttribute("media-video", "volume", THREE.Math.clamp(this.data.volume + volumeMod, 0, 1));
+      this.volumeLabel.setAttribute(
+        "text",
+        "value",
+        this.data.volume === 0 ? "MUTE" : `VOL: ${Math.round(this.data.volume * 100)}%`
+      );
+      this.volumeLabel.object3D.visible = true;
+      clearTimeout(this.hideVolumeLabelTimeout);
+      if (this.data.volume) {
+        this.hideVolumeLabelTimeout = setTimeout(() => (this.volumeLabel.object3D.visible = false), 1000);
```

Comment:
Ok to set a timeout every frame? I suppose people aren't messing with volume all the time... 

An alternative might be to write the time that the volume label should be made invisible here since tick is going to keep being called

Reply:
Yeah again, since this is only when manipulating volume I don't feel too worried about it.